### PR TITLE
[master] 07-interacting-with-pdf - add cover to pdf

### PR DESCRIPTION
**Pull request by innovia**

### CODE REVIEW

File: 07-interacting-with-pdf/cover_the_emperor.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import os
+from PyPDF2 import PdfFileReader, PdfFileWriter
```
Comment (boarik) [inline 07-interacting-with-pdf/cover_the_emperor.py:3]: `from ...` should appear before `import ...`.

Comment (innovia) [inline 07-interacting-with-pdf/cover_the_emperor.py:3]: done

**Comment by boarik** [comment]
Please assign Sveder as this PR's reviewer.

**Comment by Sveder** [comment]
Looks very good. 
Here is a list of things that repeat themselves during the previous code reviews, I'd love for you to save it and go over before you send pull request to external reviewers. I've bolded things that are relevant to this specific code review:

Rules of thumb:
Don't Repeat Yourself (DRY)
Clean code (care about the code wellness)

Additional pointers:
**Use " instead of '.**
**Use named arguments where a function expects more than 1 argument and the passed variables differ from the arguments names.**
Use descriptive variable names which fit the value that will be held by the variable (e.g. file_path instead of file).
Use try / catch with the appropriate errors.
Break long lines according to the style guide.
Think about all the possible leafs in the flow. For example, in case an `if` fails, write an `else` to handle what should happen.
Line breaks according to style guide.

**Comment by innovia** [comment]
O.K
I don't see in this PR a the use of single quote in a string

**Pull request closed without merging**